src/app/rpmostree-builtin-status.cxx

```diff
@@ -272,6 +273,24 @@ get_last_auto_update_run (GDBusConnection   *connection,
   return TRUE;
 }
 
+static gboolean
+get_update_driver_sd_unit (char **update_driver, GError **error)
```

Comment:
Should the update driver be e.g. a D-Bus property instead? That way, it's easy to query for D-Bus clients, and it's easy to add to the JSON printing path just like e.g. `transaction` so that it's also queryable via `status --json`.

Reply:
What if we just read from the state file in `/run` and add to the JSON printing path? That seems quite straightforward too. But let me know if I'm overlooking something with D-Bus properties.

Reply:
D-Bus is nice for containerized clients and apps like Cockpit which already talk to rpm-ostree over D-Bus, but given that the state file is private to rpm-ostree, we can always enhance the interface itself later on as needed.